README.md

```diff
@@ -293,11 +293,31 @@ github "YusukeHosonuma/SwiftPrettyPrint"
 
 ### Swift Package Manager
 
+Add the following line to the dependencies in your `Package.swift` file:
+
 ```swift
 .package(url: "https://github.com/YusukeHosonuma/SwiftPrettyPrint.git", .upToNextMajor(from: "1.0.0"))
 ```
 
-or add from Xcode 10+.
+Finally, include "SwiftPrettyPrint" as a dependency for your executable target:
+
+```swift
+let package = Package(
+    // name, platforms, products, etc.
+    dependencies: [
+        .package(url: "https://github.com/YusukeHosonuma/SwiftPrettyPrint.git", .upToNextMajor(from: "1.0.0")),
+        // other dependencies
+    ],
+    targets: [
+        .target(name: "<your-target-name>", dependencies: [
+            .product(name: "SwiftPrettyPrint", package: "SwiftPrettyPrint"),
+        ]),
```

Comment:
The `dependencies` can described by name when package name and library name are same.

```suggestion
        .target(name: "<your-target-name>", dependencies: ["SwiftPrettyPrint"]),
```